webhook_server_container/libs/github_api.py

```diff
@@ -759,6 +759,43 @@ def close_issue_for_merged_or_closed_pr(self, hook_action):
                 issue.edit(state="closed")
                 break
 
+    @ignore_exceptions(logger=FLASK_APP.logger)
+    def delete_remote_tag_for_merged_or_closed_pr(self, hook_action):
+        pr_tag = f"pr-{self.pull_request.number}"
+        # run regctl as a container:
+        base_regctl_command = (
+            "podman container run -i --rm --net host  -v regctl-conf:/home/appuser/.regctl/ "
```

Comment:
why do you need this `regctl-conf:/home/appuser/.regctl/` ?
you are not persist anything.

Reply:
I basically copied over the example from the readme file: https://github.com/regclient/regclient/blob/main/docs/install.md#running-as-a-container. But on trying locally without it, it looks like without it the previous authentication was ignored 
```
dbasunag@dbasunag-mac github-webhook-server % podman container run -i --rm --net host ghcr.io/regclient/regctl:latest tag ls quay.io/openshift-cnv/cnv-tests-github --verbosity debug                     
time="2024-06-24T16:23:46Z" level=debug msg="regclient initialized" VCSRef=766ee6291f882778207ff42207f9ca8b1da54e57-dirty VCSTag=v0.6.1
time="2024-06-24T16:23:46Z" level=debug msg="Listing tags" host=quay.io repository=openshift-cnv/cnv-tests-github
time="2024-06-24T16:23:46Z" level=debug msg="http req" method=GET url="https://quay.io/v2/openshift-cnv/cnv-tests-github/tags/list" withAuth=false
time="2024-06-24T16:23:47Z" level=debug msg="Auth request parsed" challenge="[{bearer map[realm:https://quay.io/v2/auth scope:repository:openshift-cnv/cnv-tests-github:pull service:quay.io]}]"
time="2024-06-24T16:23:47Z" level=debug msg="http req" method=GET url="https://quay.io/v2/openshift-cnv/cnv-tests-github/tags/list" withAuth=true
time="2024-06-24T16:23:47Z" level=debug msg="Auth request parsed" challenge="[{bearer map[realm:https://quay.io/v2/auth scope:repository:openshift-cnv/cnv-tests-github:pull service:quay.io]}]"
time="2024-06-24T16:23:47Z" level=debug msg="Failed to handle auth request" Err=unauthorized URL="https://quay.io/v2/openshift-cnv/cnv-tests-github/tags/list"
failed to list tags for quay.io/openshift-cnv/cnv-tests-github:latest: unauthorized
dbasunag@dbasunag-mac github-webhook-server % 
```

Reply:
ok